jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java

```diff
@@ -19,10 +44,18 @@ public BaseAction(Tween tween) {
         subActions.toArray(actions);
     }
 
+    /**
+     * Used to add the tween actions only if they are types of anim #{@link Action} class.
```

Comment:
"Used to" is unnecessary padding.
Describing in detail what the method does isn't very useful, particularly for a private method. Javadoc for private methods is only visible when browsing the source code. If we can't summarize what the method does at a higher level, there's no point in creating javadoc for it.